docs/userguide/008_The_Library_API.adoc

```diff
@@ -176,6 +176,24 @@ SliceAssignment legacyPackageStructure = new SliceAssignment() {
 SlicesRuleDefinition.slices().assignedFrom(legacyPackageStructure).should().beFreeOfCycles()
 ----
 
+==== Configurations
+
+There are two configuration parameters to adjust the behavior of the cycle detection.
+They can be configured via `archunit.properties` (compare <<Advanced Configuration>>).
+
+[source,options="nowrap"]
+.archunit.properties
+----
+# this will limit the maximum number of cycles to detect and thus required CPU and heap
+# default is 100
+cycles.maxNumberToDetect=50
+
+# this will limit the maximum number of dependencies to report per cycle edge
```

Comment:
A note that this affects only the output not the searching (i.e. edges will be respected but not reported) would be good to be prevent the user from increasing this number without need.

Reply:
Ah, yes, I didn't think about that :+1: I'll write this more clear

Reply:
done